core/chains/evm/client/rpc_client.go

```diff
@@ -490,6 +493,18 @@ func (r *rpcClient) SubscribeNewHead(ctx context.Context, channel chan<- *evmtyp
 	args := []interface{}{"newHeads"}
 	lggr := r.newRqLggr().With("args", args)
 
+	if r.newHeadsPollInterval > 0 {
+		interval := r.newHeadsPollInterval
+		timeout := interval
+		poller, _ := commonclient.NewPoller[*evmtypes.Head](interval, r.latestBlock, timeout, r.rpcLog)
```

Comment:
Why copy `r.newHeadsPollInterval` twice? Not a big deal either way I guess, just wondering why not just:
```
poller, _ := commonclient.NewPoller[*evmtypes.Head](interval, r.latestBlock, interval, r.rpcLog)
```
or even:
```
poller, _ := commonclient.NewPoller[*evmtypes.Head](r.newHeadsPollInterval, r.latestBlock, r.newHeadsPollInterval, r.rpcLog)
```
?

Reply:
It makes the line 499 more readable, so it's not looking like passing the same parameter twice. And it's more configurable if we decided to update the timeout value later. And it's consistent with the `SubscribeToFinalizedHeads` function